backend/mobile-backend/schema.graphql

```diff
@@ -0,0 +1,60 @@
+# This "input" configures a global authorization rule to enable public access to
+# all models in this schema. Learn more about authorization rules here: https://docs.amplify.aws/cli/graphql/authorization-rules
+input AMPLIFY { globalAuthRule: AuthRule = { allow: public } } # FOR TESTING ONLY!
+
+type GameSession @model {
+  id: ID! @primaryKey
+  game: Game! @hasOne
+  startTime: AWSDateTime
+  phaseOneTime: Int
+  phaseTwoTime: Int
+  teams: [Team] @hasMany
+  currentQuestion: Int!
+  isPhaseOne: Boolean!
+}
+
+type Game @model(subscriptions: null, timestamps: null) {
+  id: ID!
+  title: String
+  description: String
+  imageUrl: String
+  isAdvanced: Boolean
+  gameCode: Int
+  grade: String
+  questions: [Question] @hasMany
+}
+
+type Question @model(subscriptions: null, timestamps: null) {
+  id: ID!
+  question: String!
+  answer: String
+  trickAnswers: AWSJSON
+  imageUrl: String
+  instructions: AWSJSON
+  grade: String
+}
+
+type Team @model {
+  id: ID!
+  name: String!
+  gameId: ID!
+  questionId: ID!
```

Comment:
Probably won't need `gameId` here. We can also have the `questionId` to be actually the question entity